### PR TITLE
[INFINITY-2236] Explain Kafka upgrade in strict mode 

### PR DESCRIPTION
**Pull request by mbdcos**
#1449

### CODE REVIEW

**Comment by mbdcos** [comment]
ping reviewers

File: frameworks/kafka/docs/upgrade.md [hunk 1]
```diff
@@ -106,4 +110,43 @@ Since protocol and log versions have been updated with new protocol and log form
...
     
-    
+##  Upgrade in Strict Mode
+
+If you are upgrading to the DC/OS Apache Kafka package `2.0.0-0.11.0.0`  on a DC/OS cluster running in [strict security mode](https://docs.mesosphere.com/1.9/security/#security-modes), you must specify service account details during the package update process. Service account credentials enable your service to authenticate to a DC/OS cluster in strict mode.
+
+
+You can provide service account details in a JSON options file or via the DC/OS GUI.
+
+
```
Comment (sascala) [inline frameworks/kafka/docs/upgrade.md]: suggest a header and sentence edits:
### New Installation
If you are performing a fresh installation of Kafka `2.0.0-0.11.0.0` on a strict mode cluster...

File: frameworks/kafka/docs/upgrade.md [hunk 2]
```diff
@@ -106,4 +110,43 @@ Since protocol and log versions have been updated with new protocol and log form
...
+        "service": {
+            "service_account": "this_is_your_service_account_id",
+            "service_account_secret": "this_is_your_sa_secret_path"
+        }
+    }
+
+    $ docs package install  kafka --options=options.json ` --package-version=2.0.0-0.11.0.0
+
+**Note:** The syntax for specifying service account details has changed from the previous version. The `principal` and `secret_name` parameters have changed to `service_account` and `service_account_secret`. If you are upgrading your service from package version `1.1.16-0.10.1.0-beta`, you will need to specify service account details in a JSON options file.
+
```
Comment (sascala) [inline frameworks/kafka/docs/upgrade.md]: Suggest a header + edits
### Upgrade to Kafka `2.0.0-0.11.0.0`
"from `1.1.16-0.10.1.0-beta` to version `2.0.0-0.11.0.0`,"

File: frameworks/kafka/docs/upgrade.md [hunk 3]
```diff
@@ -106,4 +110,43 @@ Since protocol and log versions have been updated with new protocol and log form
...
+If you are **upgrading your existing service** in strict mode, from `1.1.16-0.10.1.0-beta` to the this new version `2.0.0-0.11.0.0`, set `service_account` and `service_account_secret` in your options.  Add service_account and service_account_secret options only in Step 1 in [Upgrade Instructions](#upgrade-kafka-from-0.10.2.x-to-0.11.0.0
+). Step 2 and Step 3 will be same. Modify Step 1 as follows if you are upgrading in `strict mode`.
```
Comment (sascala) [inline frameworks/kafka/docs/upgrade.md]: remove backpacks from `strict mode`.

**Pull request merged**